desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserActions/StorageBrowserActions.tsx

```diff
@@ -0,0 +1,92 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { Dropdown, Button } from 'antd';
+import type { MenuProps } from 'antd';
+
+import MoreVerticalIcon from '@cloudera/cuix-core/icons/react/MoreVerticalIcon';
+import InfoIcon from '@cloudera/cuix-core/icons/react/InfoIcon';
+import { i18nReact } from '../../../../utils/i18nReact';
+import { StorageBrowserTableData } from '../../../../reactComponents/FileChooser/types';
+
+import './StorageBrowserActions.scss';
+
+interface StorageBrowserActionsProps {
+  rowData: StorageBrowserTableData;
+  onSelectFile: (path: string) => void;
+  onSelectSummary: (showModal: boolean) => void;
+}
+
+const StorageBrowserActions: React.FC<StorageBrowserActionsProps> = ({
+  rowData,
+  onSelectFile,
+  onSelectSummary
```

Comment:
I meant it as selecting the summary option from the actions dropdown.

Reply:
I would skip the select part, it can be confusing. Also the  "onSelectFile" might need a better name if it is not actually called when the user is selecting a file